### ISSUE TITLE
Qr code facultatif

### ISSUE THREAD

**Issue by xgaia** [LAB-MI/attestation-deplacement-derogatoire-q4-2020#97]
Bonjour,

Comme [discuté](https://github.com/LAB-MI/deplacement-covid-19/pull/93) sur la première version de l'attestation, le QR code pose des problèmes de vie privé car le terminal servant a le scanner peux stocker ces informations.

L'idée serai de le rendre optionnel, et de le remplacer par un résumé des info de l’attestation en assez gros pour être lu à 1m de distance.

Je peux soumettre une PR avec cette fonctionnalité si besoin.

Xavier

**Comment by a2br** [comment]
Le QR code est assez pratique dans mon point de vue. Son seul problème serait de stocker la date de création sans avertissement. Je serais incapable de te dire où je l'ai lu, mais quelque part dans les issues quelqu'un avait mentionné un article où il était certifié que les infos n'étaient pas stockées. Quoi qu'il en soit, dommage que l'appli CovidReader utilisée par la police ne soit pas open source pour pouvoir s'en assurer.

**Comment by xgaia** [comment]
Je suis d'accord pour le coté pratique, c'est pour ça que je propose de le rendre optionnel pour éviter un éventuel tracking (l'appli CovidReader n'étant pas open source, il est légitime d'avoir des doutes).

Pour ceux qui décident de ne pas générer de QR code, une page avec les info en gros permettrai une lecture à distance.

**Comment by a2br** [comment]
Soit, c'est une idée à retenir.

**Comment by okpapt** [comment]
Comme il est possible de la recopier à la main ou de la remplir avec un traitement de texte, je doute fort que le QRcode soit obligatoire.
